UI/reconciliation/report.html

```diff
@@ -220,14 +261,16 @@
         name = "id_$i",
         value = row.id
     } ?>
-        <tr class="record">
-            <td><?lsmb INCLUDE input element_data = {
+        <tr class="record" id=<?lsmb "recon-line-$row.id" ?>>
+            <td><?lsmb IF !approved and !submitted;
+            INCLUDE input element_data = {
             type = "checkbox"
             name = "cleared_$row.id"
             value = row.id
             checked = row.cleared
-            }
-            ?></td>
+            "data-dojo-type"  = "lsmb/ReconciliationLine"
```

Comment:
Ah. Now I see the structure. What I'd like to see here is a structure similar to the one in the invoice lines: the elements within the line send out messages on state-change events to a line-specific channel. Interested widgets (such as the line itself) listen on the channel and act appropriately. E.g. the widget to delete the invoice line only sends out a click-event on the channel. The InvoiceLine widget takes action and removes itself both from the DOM as well as from its parent.
I'm thinking the structure of this problem is the same and it would be good to handle it the same way.